Sequencer.cs

```diff
@@ -16,39 +16,99 @@ namespace WordSequence
 {
     public class Sequencer : CustomPwGenerator
     {
-        private string GetConfigurationPath()
+        /* if returnEmptyConfig is true, or a user config file is found, returns
+         * the path to the user config file.
+         *
+         * if returnEmptyConfig is false, or the user config file path is not
+         * specified, and the user config file is not found, returns the path to
+         * the global config.
+         *
+         * Note reading the global config at startup (for a missing user config
+         * file) and then writing the user config after changing settings, allows
+         * a default configuration to be specified in the global config.
+         */

```

Comment:
I added code to support both names. I recommend keeping the old name as a deprecated feature for backwards compatibility.